constraints/src/main/java/gov/nasa/jpl/aerie/constraints/model/LinearProfile.java

```diff
@@ -58,7 +58,8 @@ public LinearProfile plus(final LinearProfile other) {
               p.valueAt(intersection.start) + o.valueAt(intersection.start),
               p.rate + o.rate
           );
-        }
+        },
+        Window.FOREVER
```

Comment:
🔻 This change was made in the first commit, but I think it belongs in the second commit.